lib/index.ts

```diff
@@ -166,6 +168,14 @@ async function main() {
 
     client.on('ready', async () => {
         console.log('Logged in');
+
+        function runner() {
+            handleFloorPriceChannel(client);
+            handleTotalVolumeChannel(client);
+        }
```

Comment:
We don't want this nested in the `.on('ready')`, take it out of this function and put it at the same level as `handleMessage`, `dispatchCommand` etc